projects/storefrontlib/src/shared/components/anonymous-consents/dialog/anonymous-consent-dialog.component.spec.ts

```diff
@@ -12,6 +12,14 @@ import { Observable, of } from 'rxjs';
 import { ModalService } from '../../modal/index';
 import { AnonymousConsentDialogComponent } from './anonymous-consent-dialog.component';
 
+@Component({
+  selector: 'cx-spinner',
+  template: `
+    <div>spinner</div>
```

Comment:
up to you, but you can keep it empty I believe

Reply:
It's a copy paste :) As the build is green, I'd keep it as is